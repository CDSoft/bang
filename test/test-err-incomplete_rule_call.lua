generator(false)

rule "foo" -- [test/test-err-incomplete_rule_call.lua:3] ERROR: rule foo: incomplete definition
