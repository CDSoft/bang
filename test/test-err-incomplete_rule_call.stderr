[test/test-err-incomplete_rule_call.lua:3] ERROR: rule foo: incomplete definition
